### PR TITLE
Mi Router compatibility test

### PR DESCRIPTION
**Pull request by RiRomain**
**Description:**
Addition of a compatibility test procedure to ensure the device is compatible with the component.

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by ludeeus — COMMENTED** [review]
This should go into the `current` branch not `next`.

File: source/_components/device_tracker.xiaomi.markdown [hunk 1]
```diff
@@ -41,3 +41,8 @@ password:
 {% endconfiguration %}
 
 See the [device tracker component page](/components/device_tracker/) for instructions how to configure the people to be tracked.
+
+### {% linkable_title Compatibility test %}
+
+To ensure that your router is compatible, navigate to http://$router_IP/api/misystem/devicelist
```
Comment (frenck) [inline source/_components/device_tracker.xiaomi.markdown]: A small suggestion to better match our documentation style.

```suggestion
To ensure that your router is compatible, navigate to `http://YOUR_ROUTER_IP/api/misystem/devicelist`.
```

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
@RiRomain Thank you for this PR!

Nevertheless, as pointed out by @ludeeus already, it targets an incorrect branch. Updates to our current documentation must target the `current` branch. Please change the target branch and rebase your changes. Please let us know if you need any assistance. 👍

**Review by fabaff — APPROVED** [review]
Thanks :bird: 

I will cherry-pick it for `current`.

**Pull request merged**